pages/post/[post].tsx

```diff
@@ -0,0 +1,149 @@
+import { fetchEntries } from "../../utils/contentfulPosts";
+import { Asset } from "contentful";
+
+interface IContent {
+    title: string
+    category: string
+    publishedDate: string
+    featuredImage: Asset
+    imageAlt: string
+    imageUrl: string
+    slug: string
+    author: string
+    body: any
+}
+
```

Comment:
Why didn't you used the `Post` interface here? That way you wouldn't had to deal with diving into `featuredImage` attributes.